include/tvm/relay/attrs/transform.h

```diff
@@ -59,6 +59,19 @@ struct ReshapeAttrs : public tvm::AttrsNode<ReshapeAttrs> {
   }
 };  // struct ReshapeAttrs
 
+/*! \brief Attributes used in squeeze operators */
+struct SqueezeAttrs : public tvm::AttrsNode<SqueezeAttrs> {
+  Array<IndexExpr> axis;
+
+  TVM_DECLARE_ATTRS(SqueezeAttrs, "relay.attrs.SqueezeAttrs") {
+    TVM_ATTR_FIELD(axis)
+        .describe("The axis to squeeze in the input tensor."
+                  "If `axis = []`, all axis get squeezed;"
+                  "Else, the dimension in axis get squeezed.")
```

Comment:
Otherwise, only the axes in `axis` get squeezed, provided they are of dimension 1.

If an axis in `axis` does not have dimension 1, _______.